dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2500,7 +2477,7 @@ def test_blockwise_parquet_annotations(tmpdir):
     assert list(layers.values())[0].annotations == {"foo": "bar"}
 
 
-def test_split_row_groups_pyarrow(tmpdir):
+def test_split_row_groups(tmpdir, engine):
```

Comment:
`split_row_groups` can/should now include the "fastparquet" engine (previously, fastparquet always used `split_row_groups=True`).